app/views/report_templates/jobs_invocations.erb

```diff
@@ -0,0 +1,35 @@
+<%#
+name: Jobs - Invocations
+snippet: false
+template_inputs:
+- name: job_id
+  required: true
+  input_type: user
+  description: ID of job to report
+  advanced: false
+  value_type: plain
+  resource_type: JobInvocation
+  hidden_value: false
+- name: hosts
+  required: false
+  input_type: user
+  description: Field for filter hosts of job invocation. Leave blank for all hosts.
+  advanced: false
+  value_type: search
+  resource_type: Host
+  hidden_value: false
+model: ReportTemplate
+-%>
+<%- invocation = load_job_invocation_by_id(input('job_id')) -%>
```

Comment:
There is still mismatch in name of the macro, does this work like this? If so, we can remove the macro from `ForemanRemoteExecution::RendererMethods`. There we have named it `find_job_invocation_by_id`

Reply:
Nice catch :+1:  I miss that :(